dockertest/config.py

```diff
@@ -85,10 +85,7 @@ def has_section(self, section):
         :param section: Name of section to check.
         :returns: True/False if section exists.
         """
-        if section == self._section:
-            return True
-        else:
-            return False
```

Comment:
R0102: 88,8: ConfigSection.has_section: The if statement can be replaced with 'return bool(test)'